ui/packages/file/src/FileUpload.svelte

```diff
@@ -44,7 +44,13 @@
 
 	let accept_file_types = "";
 	try {
-		file_types.forEach((type) => (accept_file_types += type + "/*, "));
+		for (const type of file_types) {
+			if (type.includes(".")) {
+				accept_file_types += type + ", ";
+			} else {
+				accept_file_types += type + "/*, ";
+			}
+		}
 	} catch (err) {
 		if (err instanceof TypeError) {
```

Comment:
this try/catch is weird. The backend allows the user to send either a list of strings or None (which is the default value of file_types). In the frontend, we should allow file_types to be type `file_types: string[] | null = null`. We should explicitly handle the case where file_types is null, not wrap it in this try/catch. 
If `file_types` == null, then `accept_file_types` should also equal null (not empty string), because later we set `accept={accept_file_types}` and we want to remove the `accept=` attribute entirely if we're not filtering for file types. In svelte, this is done by passing null to an attribute.